bertytech/content/blog/2FA/index.md

```diff
@@ -0,0 +1,98 @@
+
+For example, if you want to log in to your email account, you will need to present an additional method and your password. This may be a code received by SMS or code on an identification application. 
+
+Usually, the first authentication method is something you know (your password, a PIN code). The second one is information you don't know (code generated especially for authentication) but which is linked to you. 
+
+It is customary to divide into several types of information: 
+- 🔑 something you have (an object like a USB key)
+- 🐾 something you are (fingerprint, facial recognition)
+- 🧠 something you know (a password)
+- 📍 somewhere you are (a GPS location)
+
+
+> You can check on https://twofactorauth.org/ is a website/app support 2FA or not. 
+
+## 📞 Two Factor Authentication by SMS
+
+![](https://i.imgur.com/uuqGxCp.jpg)
+
+
+The second factor in identifying yourself is a code received by SMS. This is probably the method you are most familiar with. 
+
+The code received by SMS differs each time, so if your password is stolen, it won't be enough to go further. The code is linked to your cell phone number, which is a powerful security feature - unless you have been physically stolen or hacked by a [Sim Swapping](https://berty.tech/blog/sim-swapping/).
+
+A significant disadvantage of this 2FA is that if the mobile network does not work or is in a region without coverage, you will not receive the code allowing you to identify yourself. 
+Another disadvantage is if you are using connected and centralized equipment (between your tablet and mobile), you will receive the code on these devices; your login codes are as much at risk as your passwords.
+
+
+## 📱 2FA by Authentificator App:
+
+![](https://i.imgur.com/YuRNDY0.jpg)
+
+
+Authentication applications provide you with the same type of service as 2FA by SMS. However, instead of receiving the codes, they are generated locally within your smartphone or tablet. So, they offer flexibility when traveling to a location without cellular service.
+
+The operation relies on cryptographic algorithms to generate single-use passwords that integrate the notion of time (Time-based One-time Password algorithm, TOTP). 
+
+The 2FA application will first generate a start key called "seed". The latter will be derived and mixed with the time and date to make a one-time code, usually valid for 30 to 60 seconds.
+
+> 💡 A good thing to know: there are [password managers](https://berty.tech/blog/best-password-manager/) that manage TOTPs.
+
+***Open Source TOTP:***
+
+👉 https://awesomeopensource.com/projects/totp 
+ 
+👉 https://freeotp.github.io/
+
+
+The major drawback of 2FA applications is that if a scammer grabs your seed (either from your equipment or by hacking the server), he can compute any future code. If you log in and use the authentication application on the same equipment, your authentication codes are as much at risk as your passwords.
+
```

Comment:
add this one maybe -> https://krypt.co/

it's different from the others and allow very unusual workflows, i.e.: 2FA with less step than without-2FA on normal website